models/api.users.model.js

```diff
@@ -0,0 +1,8 @@
+const db = require("../db/connection");
+
+exports.fetchUsers = () => {
+  let sqlQuery = `SELECT * FROM users `;
```

Comment:
Remember to default to const, unless it is something that needs to change in the code that you have written, which in for this task is not the case